src/pages/guru/academic_guru.css

```
@@ -1,160 +1,337 @@
 }
 
+/* Content area styles */
 .academic-content {
   flex: 1;
-  padding: 2rem;
-  margin-left: 300px; /* Sesuaikan dengan lebar sidebar Anda */
-  width: calc(100% - 250px);
+  padding: 20px;
+  overflow-y: auto;
+  margin-left: 280px;
+  width: cal(100% - 280px);
```

Biome (1.9.4)

[error] Unexpected unknown function: cal

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)